### ISSUE TITLE
Screenshots?

### ISSUE THREAD

**Issue by neuromancer** [openmoh/openmohaa#26]
I think people will be more interested in testing this neat re-implementation and maybe collaborate with some code if you show the current state of this port in the README. Even early screenshots are fine!

**Comment by mohabhassan** [comment]
Hi, sorry for the late reply.
We can't really screenshot since opm client is not yet finished.
Perhaps a screenshot of opm server console would do ? it's insignificant though.
Ry,

**Comment by neuromancer** [comment]
> We can't really screenshot since opm client is not yet finished.
> Perhaps a screenshot of opm server console would do ? it's insignificant though.

I think waiting for this is perfectly fine (I don't think people will be be attracted by the console looks :smiley: )

**Comment by mohabhassan** [comment]
Agreed, but opm client will look exactly identical to AA client unless we make something point out otherwise.

**Comment by neuromancer** [comment]
> Agreed, but opm client will look exactly identical to AA client unless we make something point out otherwise.

IMHO, running MoHAA natively in platforms like Linux is a great thing to show.

**Comment by mohabhassan** [comment]
> 
> 
> > Agreed, but opm client will look exactly identical to AA client unless we make something point out otherwise.
> 
> IMHO, running MoHAA natively in platforms like Linux is a great thing to show.

Indeed! I didn't think about when I replied before.
We could also start looking into android ports, once everything on win/nix/macos/osx is stable.

**Comment by neuromancer** [comment]
It seems we already have some screenshots, great work!

**Comment by smallmodel** [comment]
Closing as there is now a screenshot
